.prettierrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
Did this file need to have the JS extension as part of this work? I think on my setup it runs fine without it, but I don't know if there's any kind of tradeoff to include/exclude the JS extension.

FWIW, it looks like both design-system and HCgov use the JS extension. So I imagine this change was for consistency only

Reply:
I'm unsure if we have to have the extension or not, I was going oof of what was already in the core and HCgov design system